components/gc-intro/gc-intro-block-en.html

```diff
@@ -0,0 +1,88 @@
+---
+title: Introduction block pattern
+description: Documentation and working example for the introduction block pattern
+language: en
+altLangPage: gc-intro-block-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+	<p>The introduction block pattern introduces the content of a landing page.</p>
+
+	<h2>On this page</h2>
+	<ul>
+		<!-- <li><a href="#inc-components">Included components</a></li> -->
+		<li><a href="#before-start">Before you start</a></li>
+		<li><a href="#accessibility-guidance">Accessibility guidance</a></li>
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+
+	<h2 id="before-start">Before you start</h2>
+	<p>The introduction block pattern is a mandatory element for institutional landing pages and topic pages. It’s optional for other landing pages.</p>
+
+	<ul>
+		<li><a href="">How to set up the institutional landing page</a></li>
+		<li><a href="">Introduction block pattern</a></li>
+	</ul>
+
+	<h2 id="accessibility-guidance">Accessibility guidance</h2>
+	<p>List of accessibility guidances to be followed to ensure component is used in respect of the accessibility guideline and EN 301 549 standard (including WCAG 2.1 Level AA).</p>
+	<p>There is <span>no accessibility guidance</span> defined.</p>
+
+	<h2 id="variations">Variations</h2>
+	<p>The introduction block pattern consists of:</p>
+	<ul>
+		<li>title</li>
+		<li>short description</li>
+	</ul>
+	<p>It can also include:</p>
+	<ul>
+		<li>image</li>
+		<li>supertask button</li>
+	</ul>
+
+	<h3>Introduction block</h3>
+	<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
+		<div class="container">
+			<div class="row mrgn-bttm-md">
+				<div class="col-md-7">
+				<h1 property="name" id="wb-cont">[Institution name]</h1>
+					<p>Short description of the institution’s mandate.</p>
+					<a class="btn btn-call-to-action" href="#">Super task button [optional]</a>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div class="container">
+		<h3>Code</h3>
+		<pre><code>
+		&lt;div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w"&gt;
+	&lt;div class="container"&gt;
+		&lt;div class="row mrgn-bttm-md"&gt;
+			&lt;div class="col-md-7"&gt;
+				&lt;h1 property="name" id="wb-cont"&gt;[Institution name]&lt;/h1&gt;
+				&lt;p&gt;Short description of the institution’s mandate.&lt;/p&gt;
+				&lt;a class="btn btn-call-to-action" href="#"&gt;Super task button [optional]&lt;/a&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/div&gt;
+	</code></pre>
+</div>
+
```

Comment:
Should we also add the introduction block pattern that contain the by-line?

ref. https://design.canada.ca/common-design-patterns/institutional-byline.html

Reply:
The byline is being moved to the bottom of the page, like how it is in the theme and topic beta, so it doesn't make sense as part of this component.

Reply:
Good to know, then my comment is not applicable then.